### DISCUSSION TITLE
Custom Startup Theme

### DISCUSSION THREAD

**Discussion by LeNitrous** [ppy/osu#13057] [Ideas]
**Describe the new feature:** Allow skins to change the startup theme. This has been discussed before in Discord when I was doing https://github.com/ppy/osu/pull/6219 and is also a featured request as stated in [this forum thread](https://osu.ppy.sh/community/forums/topics/244938). Creating an issue for visibility.

**Proposal designs of the feature:** Include a single difficulty beatmapset to skins and use it as the startup theme. It can be a select-able option between Triangles and Circles.

[Here's a quick example video](https://streamable.com/vqnmc)

**Comment by floomster** [comment]
you beat me to this 😕😕

**Comment by peppy** [comment]
As mentioned, we're still a bit far off from adding new skinnable features.

**Comment by fakeaireuonboard** [comment]
I'd like to look forward on this feature request. It has been 5 years since the last update regarding to the request. so is it possible, that this feature will be added in the near future? Thanks.
